README.md

```diff
@@ -76,12 +71,6 @@ The majority of policies target a specific type of Kubernetes resource, like
 Pod, Ingress, Service and similar. Because of that, another possible approach
 is to unmarshal the incoming object into a native Go type.
 
-Because of the current TinyGo limitations, both the usage of the `encoding/json`
-package and the usage of the official Kubernetes Go types defined
-under the `k8s.io` pacakges (e.g. `k8s.io/api/core/v1`) is not possible.
```

Comment:
This is still true, can you leave this lines and maybe integrate them with the `README.md` that you just added to the k8s-objects generated files?

Reply:
this was already mentioned in the README generated file: 

https://github.com/kubewarden/k8s-objects-generator/blob/0c689b838bebe2acae0424bdbe5c6613f71e981b/object_templates/README.md.tmpl#L18

Reply:
i've copied this paragraph over here instead